java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/UserJarsST.java

```diff
@@ -76,6 +77,20 @@ void shouldApplyTableIteratorFromUserJarDuringIngest(SleeperSystemTest sleeper)
                 .containsExactlyInAnyOrderElementsOf(sleeper.generateNumberedRows(LongStream.range(50, 100)));
     }
 
+    @Test
+    void shouldApplyTableIteratorFromPropertiesDuringIngest(SleeperSystemTest sleeper) throws Exception {
```

Comment:
This file is for testing applying user jars, which can only really be done in a deployed instance.

I don't think these new tests belong here? I don't think they need to be done as system tests, and it's extremely slow to run them this way.